cmake/modules/FindJsonSchemaBuilder.cmake

```diff
@@ -10,8 +10,13 @@
 if(NOT TARGET JsonSchemaBuilder::JsonSchemaBuilder)
   if(KODI_DEPENDSBUILD OR CMAKE_CROSSCOMPILING)
     add_executable(JsonSchemaBuilder::JsonSchemaBuilder IMPORTED GLOBAL)
-    set_target_properties(JsonSchemaBuilder::JsonSchemaBuilder PROPERTIES
-                                                               IMPORTED_LOCATION "${NATIVEPREFIX}/bin/JsonSchemaBuilder")
+    if(CORE_SYSTEM_NAME STREQUAL windows OR CORE_SYSTEM_NAME STREQUAL windowsstore)
```

Comment:
@wsnipex windowsstore is the predefined string to use to build for Windows store, it's documented in the cmake docs under the cross compile section